source/Ship.cpp

```diff
@@ -239,13 +239,20 @@ void Ship::Load(const DataNode &node)
 				if(child.Size() >= 2)
 					outfit = GameData::Outfits().Get(child.Token(1));
 			}
-			Angle gunPortAngle = Angle(0.);
+			vector<Angle> angles{};
 			bool gunPortParallel = false;
 			if(child.HasChildren())
 			{
 				for(const DataNode &grand : child)
 					if(grand.Token(0) == "angle" && grand.Size() >= 2)
-						gunPortAngle = grand.Value(1);
+					{
+						const int end = min(4, grand.Size());
+						for(int i = 1; i < end; ++i)
+							angles.push_back(grand.Value(i));
```

Comment:
I feel that a separate keyword for the turn-range will be more clear;
`turnRange <min> <max>` => for specifying the allowed turnrange for turrets (and maybe also for guns?)

Reply:
"turnRange", "turn range" or "angle of traverse". I'll use what you like.
And I'll create "idle angle", "default angle", or something.  I would like to know your suggestion.

Reply:
My preference for the keyword is `turn range`, but I'm not a native English speaker, so you might want to get some input from other reviewers for this choice.